pkg/cmd/container/run_mount.go

```diff
@@ -72,6 +73,13 @@ func withMounts(mounts []specs.Mount) oci.SpecOpts {
 		}
 
 		s.Mounts = append(s.Mounts, mounts...)
+
+		sort.Slice(s.Mounts, func(i, j int) bool {
+			// Consistent with the less function in Docker.
+			// https://github.com/moby/moby/blob/0db417451313474133c5ed62bbf95e2d3c92444d/daemon/volumes.go#L45
```

Comment:
nit: maybe put a link to `Less` instead of `parts` since `Less` is called out in the comment.

https://github.com/moby/moby/blob/24.0/daemon/volumes.go#L34

Reply:
Fixed.